common/workunit/workunit.cpp

```diff
@@ -1494,7 +1494,14 @@ class GraphScopeIterator : public CInterfaceOf<IConstWUScopeIterator>
         {
             if (whichProperties & PTattributes)
             {
-                playAttribute(visitor, WaLabel);
+                Owned<IAttributeIterator> nodeAttrs = cur.getAttributes();
```

Comment:
General implementation that covers label and new balanced attribute

Reply:
Other than label and the new balanced attribute, does this change mean that other attributes will be returned too?